opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/EventRestService.java

```diff
@@ -67,6 +69,9 @@
     @Autowired
     private EventDao m_eventDao;
 
+    @Autowired
+    private EventIpcManager m_eventForwarder;
```

Comment:
You might change this to be an EventForwarder type instead of the full EventIpcManager super-interface.